From 26335cba9b8ef877ea020b240a35336cfb3a6c40 Mon Sep 17 00:00:00 2001
From: mGaik <mahegaik@gmail.com>
Date: Mon, 3 Jul 2023 13:33:38 +0000
Subject: [PATCH] First Patch

---
 userprog.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/userprog.c b/userprog.c
index a400e9a..1d0fc18 100644
--- a/userprog.c
+++ b/userprog.c
@@ -3,5 +3,6 @@
 int main()
 {
 	printf("Hello Cmake\n");
+	printf("Hello Cmake2\n");
 	return 0;
 }
-- 
2.25.1

